python_scripts/verify_mbox.py

```diff
@@ -0,0 +1,49 @@
+#! /usr/bin/env python3
+"""Verify that an MBOX file contains all the messages in another MBOX file.
```

Comment:
To clarify either change this line (or add a new line to the docstring) clarifying that we chack that all messages in the first file are present in the second file (i.e. the order in which the files are provided matters).

Reply:
done